docs/develop/api-reference/response-objects.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 5
+sidebar_label: Response objects
+title: Response object API reference
+description: Learn how to interact with the response object in the Momento API.
+slug: /develop/api-reference/response-objects
+---
+
+# Response objects
+
+These are the baseline responses for all commands. Some commands will add more data and functionality.
+
+Commands fall *generally* into two categories. Those which respond with Success or Error and those which respond with Hit, Miss, or Error.
+
+### Response
```

Comment:
I would remove this entire `### Response` section.  This is internal sausage-making stuff.  Just skip straight to the `Error` part.